docs/pages/enroll-resources/mcp-access/integration-guides/github.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: GitHub
+description: Set up a GitHub MCP server for access through Teleport
+tags:
+- how-to
+- mcp
+- zero-trust
+---
+
+This guide demonstrates how to run a GitHub MCP server and connect it via
+Teleport.
+
+## How it works
+
+(!docs/pages/includes/mcp-access/integration-how-it-works-service-token.mdx serviceName="GitHub" token="a personal access token of a service account"!)
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx edition="Teleport (v18.3.0 or higher)" clients="\`tsh\` client"!)
+- Access to a service account to your GitHub organization.
+- A host to run the MCP server that is reachable by the Teleport Application Service.
+- Running [Teleport Application Service](../getting-started.mdx).
+- A Teleport user with sufficient permissions (e.g. role `mcp-user`) to access
+  MCP servers.
+
+## Step 1/3. Create a personal access token
+
+Log in to GitHub using your service account. Navigate to Settings > Developer
+Settings > Personal access tokens, then click **Generate new token**.
+
+When creating the token, grant only the minimal permissions needed. Avoid broad
+scopes such as write or admin access unless absolutely required.
+
+Once the token is created, save it for use in the next step.
+
+## Step 2/3 Run the GitHub MCP server
+
+First, install [`mcp-proxy`](https://github.com/sparfenyuk/mcp-proxy) on the
+host that will run the MCP server:
+```code
+# Option 1: With uv (recommended)
+$ uv tool install mcp-proxy
+
+# Option 2: With pipx (alternative)
+$ pipx install mcp-proxy
+```
+
+Now start the [GitHub MCP server](https://github.com/github/github-mcp-server)
```

Comment:
I would move this link to "How it works", where we first introduce the GitHub MCP server.